modules/gapi/src/backends/fluid/gfluidimgproc.cpp

```diff
@@ -788,8 +788,6 @@ GAPI_FLUID_KERNEL(GFluidGaussBlur, cv::gapi::imgproc::GGaussBlur, true)
                               Buffer&    dst,
                               Buffer&    scratch)
     {
-        GAPI_Assert(ksize.height == 3);
```

Comment:
Complete removal of this assert tells that the kernel now can work with arbitrary kernel size, but there is a hardcoded buffSize = 5 below, so it will not work with kernel sizes > 5

Reply:
Applied.